docs/python_docs/python/api/gluon/nn/index.rst

```diff
@@ -36,46 +36,24 @@ We group all layers in these two modules according to their categories.
 
 .. currentmodule:: mxnet.gluon
 
-Blocks
-------
-
-.. autosummary::
-   :nosignatures:
-   :toctree: .
-
-   nn.Block
-   nn.HybridBlock
-   nn.SymbolBlock
 
 
 Sequential containers
 ---------------------
 
 
 .. autosummary::
-    :toctree: _autogen
     :nosignatures:
```

Comment:
This was questioned in another issue. What are some set with `:nosignatures:` and others are not?

Reply:
This is an aesthetic/readability choice and the idea here is similar to the comment in the other issue. Here the doc is listing different classes under the different groupings so there's no need to have the method signatures.

At the bottom when autosummary generates an alphabetical listing of the classes, the method signatures are included.